src/frontend/screens/ObservationEdit/SaveButton.tsx

```diff
@@ -184,11 +184,12 @@ export const SaveButton = ({
       return;
     }
 
-    const hasLocation = value.lat && value.lon;
+    const hasLocation = value.lat !== undefined && value.lon !== undefined;
     const locationSetManually = value.metadata.manualLocation;
     if (
-      locationSetManually ||
-      (hasLocation && isGpsAccurate(value.metadata.position?.coords?.accuracy))
+      hasLocation &&
+      (locationSetManually ||
+        isGpsAccurate(value.metadata.position?.coords?.accuracy))
```

Comment:
potentially subtle bug fix? noticed that it differs from Mapeo Mobile implementation:

https://github.com/digidem/mapeo-mobile/blob/0c0ebbb9ef2261e21cd1d1c8bd5ab2fe42017ea3/src/frontend/screens/ObservationEdit/SaveButton.js#L85